### PR TITLE
Add option to scrap archive builds

### PR DESCRIPTION
**Pull request by Victor-IX**
Add the option to scrap archive builds from the different archive JSON API files 

image

### CODE REVIEW

File: source/threads/scraper.py [hunk 1]
```diff
@@ -171,7 +174,20 @@ def get_download_links(self):
...
+        branches = [
+            "daily/archive" if get_show_daily_archive_builds() and branch == "daily" else branch for branch in branches
+        ]
+        branches = [
+            "experimental/archive" if get_show_experimental_archive_builds() and branch == "experimental" else branch
+            for branch in branches
+        ]
+        branches = [
+            "patch/archive" if get_show_patch_archive_builds() and branch == "patch" else branch for branch in branches
+        ]
```
Comment (zeptofine) [inline source/threads/scraper.py:188]: This could probably be done using a single tuple instead of multiple for loops

**Comment by zeptofine** [comment]
Nice, i was unaware this was a thing

File: source/widgets/settings_window/blender_builds_tab.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from requests import get
```
Comment (zeptofine) [inline source/widgets/settings_window/blender_builds_tab.py:1]:
```suggestion
```
While `requests` is a secondary dependency, it's not used in the code and only exists if you install the repo in dev mode

**Review by zeptofine — CHANGES_REQUESTED** [review]
Nice, i was unaware this was a thing

File: source/threads/scraper.py (hunk 1 above)
Comment (Victor-IX) [reply source/threads/scraper.py]: I knew there was a better way to do it 😅

**Pull request merged**